interpreter/src/main/scala/pl/touk/nussknacker/engine/compiledgraph/node.scala

```diff
@@ -40,9 +40,11 @@ object node {
 
   case class EndingCustomNode(id: String, ref: String) extends Node
 
-  case class SubprocessStart(id: String, params: List[Parameter], next: Next) extends Node
+  case class SubprocessUsageStart(id: String, params: List[Parameter], next: Next) extends Node
```

Comment:
Maybe we can rename it to Fragment if we touching this class?

Reply:
:+1: